bitnami/argo-workflows/templates/server/ingress.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: {{ include "common.capabilities.ingress.apiVersion" . }}
+kind: Ingress
+metadata:
+  name: {{ include "argo-workflows.server.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: server
+    app.kubernetes.io/part-of: argo-workflows
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if or .Values.ingress.annotations .Values.commonAnnotations .Values.ingress.certManager }}
+  annotations:
+    ingress.kubernetes.io/rewrite-target: /$2
```

Comment:
Could you add a small comment explaining this? Not sure if the upstream one provides an explanation